src/debugger/nodeDebugWrapper.ts

```diff
@@ -135,8 +136,16 @@ Telemetry.init("react-native-debug-adapter", version, true).then(() => {
             args.target || "simulator"
         ];
 
+        if (!isNullOrUndefined(args.logCatArguments)) { // We add the parameter if it's defined (adapter crashes otherwise)
+            args.args = args.args.concat([args.logCatArguments]);

```

Comment:
I've just got it! Nice catch! :)

I've fixed it by transforming the array into a string.